.travis.yml

```diff
@@ -3,8 +3,19 @@ node_js: "8"
 dist: trusty
 cache:
   yarn: true
+  directories:
+  - "$HOME/gcloud"
 install:
   - yarn
+  - if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then mkdir -p $HOME/gcloud && wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud && cd $HOME/gcloud && tar xzf google-cloud-sdk.tar.gz && printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh && cd $TRAVIS_BUILD_DIR;
+    fi
+  - ${HOME}/gcloud/google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator
+before_script:
+  - ${HOME}/gcloud/google-cloud-sdk/bin/gcloud beta emulators datastore start --project bot-render --quiet &
+  - sleep 3 # wait for datastore emulator to start
```

Comment:
You might be able to do this instead of sleeping, `-m1` tells grep to exit after the first match:

`(${HOME}/gcloud/google-cloud-sdk/bin/gcloud beta emulators datastore start --project bot-render --quiet &) | grep -m1 "now running"`